### PR TITLE
Fix control flow issues, which occurs during reconnect

### PR DESCRIPTION
**Pull request by dynbit**

### CODE REVIEW

File: VelocitySDK/Core/VLTRecordingConfig.m [hunk 1]
```diff
@@ -25,8 +25,8 @@ - (nonnull instancetype)initWithHandshakeResponse:(VLTPBHandshakeResponse *)hand
 {
     self = [super init];
     if (self) {
-        _sampleSize        = handshakeResponse.sampleSize;
-        _captureInterval   = handshakeResponse.captureInterval;
+        _sampleSize        = 6; // handshakeResponse.sampleSize;
```
Comment (Antanas) [inline VelocitySDK/Core/VLTRecordingConfig.m:28]: sure?

File: VelocitySDK/VLTClient.m [hunk 2]
```diff
@@ -88,21 +90,34 @@ - (void)initializeWsApiClient
...
+    [self startMotionSensing];
+}
+
 - (void)closeWsApiClientIfNeeded
 {
     if (self.wsApiClient && !self.wsApiClient.isClosed) {
+        [self.wsApiClient setOnOpen:nil];
+        [self.wsApiClient setOnClose:nil];
+        [self.wsApiClient setOnError:nil];
+        [self.wsApiClient setOnPong:nil];
```
Comment (Antanas) [inline VelocitySDK/VLTClient.m:120]: Why are these needed? If the callback blocks doesn't retain variables these shouldn't be necessary?

**Review by Antanas — CHANGES_REQUESTED** [review]

File: VelocitySDK/Core/VLTRecordingConfig.m (hunk 1 above)
Comment (dynbit) [reply VelocitySDK/Core/VLTRecordingConfig.m:28]: For now we need to have it configured `6s` and `6s`, later on we will have in controllable from API side.

File: VelocitySDK/VLTClient.m (hunk 2 above)
Comment (dynbit) [reply VelocitySDK/VLTClient.m:120]: As there a time period between `close` and actual object deallocation, but in that we are not interesting into those blocks anymore. But maybe I can move those to `VLTWsApiClient` it self

**Review by Antanas — APPROVED** [review]
